packages/office-ui-fabric-react/src/components/Rating/Rating.tsx

```diff
@@ -44,106 +70,72 @@ export class Rating extends BaseComponent<IRatingProps, IRatingState> {
   }
 
   public render() {
-    let stars: JSX.Element[] = [];
-    for (let i = this.props.min as number; i <= (this.props.max as number); ++i) {
-      stars.push(this._renderStar(i));
+    let stars = [];
+
+    for (let i = this.props.min as number; i <= (this.props.max as number); i++) {
+      let ratingStarProps: IRatingStarProps = {
+        fillPercentage: this._getFillingPercentage(i),
+        disabled: this.props.disabled ? true : false
+      };
+
+      stars.push(
+        <button
+          className={ css('ms-Rating-button', styles.ratingButton, {
+            ['ms-Rating--large ' + styles.rootIsLarge]: this.props.size === RatingSize.Large,
+            ['ms-Rating--small ' + styles.rootIsSmall]: this.props.size !== RatingSize.Large
+          }) }
+          key={ i }
+          { ...((i === Math.ceil(this.state.rating as number)) ? { 'data-is-current': true } : {}) }
+          onFocus={ this._onFocus.bind(this, i) }
+          disabled={ this.props.disabled || this.props.readOnly ? true : false }
+          role='presentation'
```

Comment:
role='presentation', why?

Reply:
screen is reading out as button whenever the focus moves to that. So avoid that we marked this as presentation only. Also, same thing we discussed during our last meeting